src/main/java/uk/gov/hmcts/reform/coh/service/QuestionService.java

```diff
@@ -88,8 +97,8 @@ public void deleteQuestion(Question question) {
 
     public Question updateQuestion(Question currentQuestion, Question updateToQuestion){
         QuestionState proposedState = updateToQuestion.getQuestionState();
-        QuestionState issuedState = questionStateService.retrieveQuestionStateById(QuestionState.ISSUED);
-        if(proposedState.equals(issuedState)) {
+        Optional<QuestionState> issuedState = questionStateService.retrieveQuestionStateByStateName(QuestionStates.ISSUED.getStateName());
+        if(proposedState.equals(issuedState.get())) {
```

Comment:
SonarQube requires that issuedState,isPresent() is called before accessing the value.